tests/data/conda_format_repo/linux-64/current_repodata.json

```diff
@@ -27,10 +13,10 @@
       "legacy_bz2_md5": "2e5b81671cc2b53177b94b258b1658b2",
       "license": "zlib",
       "license_family": "Other",
-      "md5": "2e5b81671cc2b53177b94b258b1658b2",
+      "md5": "cb23839f06da92ec92c6d2d2d5e0cb4a",
       "name": "zlib",
-      "sha256": "1f07873d6aa0cd18760a7b215d9956f7fa63c2f2761b03f293c659fe982923fa",
-      "size": 122456,
+      "sha256": "f33afff2144e1d6c4cf2e3e184937db8bf825b4537df74662aea5ee207b6a31b",
```

Comment:
These hashes didn't match the test repository contents.

We'll see if any tests require the missing `libgcc-ng`.